packages/core/src/browser/dialogs/dialogs.ts

```diff
@@ -0,0 +1,109 @@
+/********************************************************************************
+ * Copyright (C) 2020 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
```

Comment:
I don’t understand separation. This file does not have any dialogs only props for dialogs and overlay service. It does not follow our file naming conventions as well, there a file name should follow to of main exported type.

Reply:
Are you talking about the dialog separation in general, or just `dialogs` and `abstract-dialog`? The latter, I have no opinion on, I don't really understand everything here yet. But the former is because that file was getting really crowded and there was no good place to put the react dialog unless that file should just be larger and larger.

Reply:
Is this resolved by the latest merge of `abstract-dialogs.ts` and `dialogs.ts`?

Reply:
yes, thanks